README.md

```diff
@@ -31,11 +31,56 @@ git remote set-branches --add origin [remote-branch]
 git fetch
 ```
 
+## Databases
+
+We have also generated tiny versions of databases for tools that normally require very large databases
+
+### CAT_pack
+
+For CAT of CAT_pack we did the following, and using CAT_pack (v6.0):
+
+```bash
+## Download the FASTA of coding sequences FASTA AA of B. fragilis
+curl "https://www.ncbi.nlm.nih.gov/sviewer/viewer.cgi?tool=portal&save=file&log$=seqview&db=nuccore&report=fasta_cds_aa&id=1992822979&extrafeat=null&conwithfeat=on&hide-cdd=on&ncbi_phid=CE8C15326D6BB8C10000000006490560" -o sequence.txt
+
+## use my scripts to filter NCBI nodes/names/acc2taxid files to just a given taxid (basically fancy iterative greps)
+bash ~/bin/taxdmp_filter.sh 817 ## for nodes/naames
+bash ~/bin/accession2taxid_filter.sh 817 ## for accessions
+
+## Repair headers of B. fragilis protein FASTA so the NCBI accession can be read by CAT properly
+sed 's/lcl|//g;s/_/ /2' sequence.txt > sequence_fixedheaders.txt
+
+## Generate database using CAT_pack prepare
+CAT_pack prepare --db_fasta input_files/sequence_fixedheaders.txt --names input_files/names_reduced.dmp --nodes  input_files/nodes_reduced.dmp --acc2tax input_files/accession2taxid_reduced.dmp --db_dir test/
+
+## Test using uncompressed contigs from metaspades assembly (note: --no_stars was required for some reason but seesms to only occur when we have a single genome in there possible..)
```

Comment:
Will this be a problem using it when testing in #mag? Or will it be OK to just add `--no-stars` to the test config?

Reply:
The latter as far as I can tell :)

The flag was added for this purpose according to GitHub

Reply:
afaik Maxbin2 will force a split into the contigs, it just assumes there are 2+ genomes (at least that happened a few years ago). But thats fine I think.